src/handlers/transfers/handler.js

```diff
@@ -39,30 +39,32 @@
 
 const Logger = require('@mojaloop/central-services-shared').Logger
 const TransferService = require('../../domain/transfer')
-const Utility = require('../lib/utility')
+const Util = require('../lib/utility')
 const Kafka = require('../lib/kafka')
 const Validator = require('./validator')
-const TransferState = require('../../lib/enum').TransferState
-const TransferStateEnum = require('../../lib/enum').TransferStateEnum
 const Enum = require('../../lib/enum')
-const TransferEventType = Enum.transferEventType
-const TransferEventAction = Enum.transferEventAction
+const TransferState = Enum.TransferState
+const TransferStateEnum = Enum.TransferStateEnum
+const TET = Enum.transferEventType
+const TEA = Enum.transferEventAction
 const TransferObjectTransform = require('../../domain/transfer/transform')
 const Errors = require('../../lib/errors')
 const Metrics = require('@mojaloop/central-services-metrics')
 const Config = require('../../lib/config')
-
-// TODO: This errorCode and errorDescription are dummy values until a rules engine is established
-const errorGenericCode = 3100
-const errorGenericDescription = Errors.getErrorDescription(errorGenericCode)
-const errorModifiedReqCode = 3106
-const errorModifiedReqDescription = Errors.getErrorDescription(errorModifiedReqCode)
-const errorInternalCode = 2001
-const errorInternalDescription = Errors.getErrorDescription(errorInternalCode)
-const errorTransferExpCode = 3300
-const errorTransferExpDescription = Errors.getErrorDescription(errorTransferExpCode)
-const errorTransferIdNotFoundCode = 3208
-const errorTransferIdNotFoundDescription = Errors.getErrorDescription(errorTransferIdNotFoundCode)
+const Uuid = require('uuid4')
+
+const eEnum = {
```

Comment:
would it not be better to store this enum in under lib/errors

Reply:
ok